sdk/resourcemanager/azure-resourcemanager-redis/src/main/java/com/azure/resourcemanager/redis/models/RedisCache.java

```diff
@@ -564,7 +565,10 @@ enum RedisVersion {
         V6("6"),
         /**
          * version 4.x.x
+         * @deprecated Because Redis version 4 is no longer supported by the open source community,
+         *             it will be retired from Azure Cache for Redis.
          */
+        @Deprecated
         V4("4");
```

Comment:
A warning here, adding value to enum when it is used only in input is fine. But adding value when the enum is used in output is considered breaking change.

Reply:
Got it. The extra method will introduce breaking change when new versions are introduced. Will remove it.